### PR TITLE
chore(package.json): update angular.js to v1.9.2 along with other non-angularjs related dependencies

### PR DESCRIPTION
**Pull request by jzolnowski**
After executing `npm install`, `package-lock.json` was also updated to reflect the changes from `package.json`.
In the process of updating the lockfile, several versions of the dependencies were upgraded automatically.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,11 +37,11 @@
     "@angular/upgrade": "16.1.2",
     "@schuchard/prettier": "^5.1.0",
     "angular": "npm:@xlts.dev/angular@1.9.1",
```
Comment (gkalpak) [inline package.json]: This also needs updating.

**Review by gkalpak — CHANGES_REQUESTED** [review]
We should also update the lockfile.

File: package.json (hunk 1 above)
Comment (jzolnowski) [reply package.json]: Done. Lock file updated as well

**Review by gkalpak — CHANGES_REQUESTED** [review]
This PR now contains more updates than just _"angular.js to v1.9.2"_.
We should either limit the changes to AngularJS or update the commit message to reflect the actual changes.

**Comment by jzolnowski** [comment]
Commit message updated

**Comment by gkalpak** [comment]
What do you mean by _"angular.js-based libraries"_?

**Comment by jzolnowski** [comment]
> What do you mean by _"angular.js-based libraries"_?

All the rest angular.js based packages like:
```
 "angular-animate": "npm:@xlts.dev/angular-animate@^1.9.2",
 "angular-aria": "npm:@xlts.dev/angular-aria@^1.9.2",
 "angular-material": "npm:@xlts.dev/angular-material@^1.2.4",
 "angular-messages": "npm:@xlts.dev/angular-messages@^1.9.2",
 "angular-sanitize": "npm:@xlts.dev/angular-sanitize@^1.9.2",
 ```

**Review by gkalpak — CHANGES_REQUESTED** [review]
>> What do you mean by _"angular.js-based libraries"_?
>
> All the rest angular.js based packages like:
> ```
>  "angular-animate": "npm:@xlts.dev/angular-animate@^1.9.2",
>  "angular-aria": "npm:@xlts.dev/angular-aria@^1.9.2",
>  "angular-material": "npm:@xlts.dev/angular-material@^1.2.4",
>  "angular-messages": "npm:@xlts.dev/angular-messages@^1.9.2",
>  "angular-sanitize": "npm:@xlts.dev/angular-sanitize@^1.9.2",
>  ```

I see. I don't think we need to explicitly mention these libraries in the commit message, since they are part of the AngularJS framework and can be collectively referred to as AngularJS. 

But this PR also updates other non-AngularJS related dependencies. We should either mention that in the commit message or move those other updates to a separate commit/PR.

**Comment by jzolnowski** [comment]
> But this PR also updates other non-AngularJS related dependencies. We should either mention that in the commit message to move those other updates to a separate commit/PR.

It's mentioned by `Also update lockfile.` commit part 😄

**Comment by gkalpak** [comment]
>> But this PR also updates other non-AngularJS related dependencies. We should either mention that in the commit message to move those other updates to a separate commit/PR.
>
> It's mentioned by `Also update lockfile.` commit part 😄

Not sure if this is supposed to be a serious answer or a joke (given the 😄 at the end) 😅, but (just in case):
a. _"update lockfile"_ does not really give any info on what/why/how it was updated and does not necessarily imply that dependencies were upgraded and
b. there is no reason to not mention this in the commit message header and bury it in the body.

**Review by gkalpak — APPROVED** [review]
Awesome 😍
Thank you!

**Pull request merged**